modules/compliance-supported-profiles.adoc

```diff
@@ -162,43 +325,147 @@ The Compliance Operator provides the following compliance profiles:
  `s390x`
 |
 
-|ocp4-pci-dss-node
-|PCI-DSS v3.2.1 Control Baseline for Red Hat OpenShift Container Platform 4
+|ocp4-pci-dss-3-2 ^[3]^
+|PCI-DSS v3.2.1 Control Baseline for {product-title} 4
+|Platform
+|
+|link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI Security Standards &#174; Council Document Library]
+|`x86_64`
+|
+
+|ocp4-pci-dss-4
+|PCI-DSS v4 Control Baseline for {product-title} 4
+|Platform
+|1.6.0+
+|link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI Security Standards &#174; Council Document Library]
+|`x86_64`
+ `ppc64le`
+ `s390x`
+|
+
+|ocp4-pci-dss-node ^[1]^
```

Comment:
We have a gap with pci-dss v4 on power. We're still working on the Security Profiles Operator Enablement.